src/libecalc/core/consumers/compressor/component.py

```diff
@@ -84,15 +81,11 @@ def evaluate(
             compressor = self._temporal_model.get_model(timestep)
             operational_settings_for_timestep = operational_settings.get_subset_for_timestep(timestep)
             evaluated_timesteps.extend(operational_settings_for_timestep.timesteps)
-            if isinstance(compressor, VariableSpeedCompressorTrainCommonShaftMultipleStreamsAndPressures):
-                raise NotImplementedError("Need to implement this")
-            elif issubclass(type(compressor), CompressorModel):
-                model_result = compressor.evaluate_rate_ps_pd(
-                    rate=np.asarray(total_requested_inlet_stream.rate.values),
-                    suction_pressure=np.asarray(total_requested_inlet_stream.pressure.values),
-                    discharge_pressure=np.asarray(operational_settings_for_timestep.outlet_stream.pressure.values),
-                )
-                model_results.append(model_result)
+            model_result = compressor.evaluate_streams(
+                inlet_streams=operational_settings.inlet_streams,
```

Comment:
streams are already mixed above in `total_requested_inlet_stream`. Should that be removed then?